nBTC/sources/nbtc.move

```diff
@@ -162,9 +162,9 @@ public fun mint(
     });
 }
 
-public fun burn(treasury: &mut WrappedTreasuryCap, coin_to_burn: Coin<NBTC>, _ctx: &mut TxContext) {
-    //TODO: implement logic to guard burning
-    coin::burn(&mut treasury.cap, coin_to_burn);
+public fun redeem(treasury: &mut WrappedTreasuryCap, coins: vector<Coin<NBTC>>, _ctx: &mut TxContext) {
```

Comment:
what do you think about using a vector here?

Reply:
Hm, it def doesnt hurt, on the other hand we do not represent nBTC as UTXOs so its not critical.

Reply:
this is still a proposal, we will decide about the final interface when implementing it.